cmd/manager/main.go

```diff
@@ -123,20 +123,15 @@ func init() {
 		"Duration representing how long before expiration TLS certificates should be reissued",
 	)
 	Cmd.Flags().Bool(
-		AutoInstallWebhooksFlag,
+		SetupWebhookCertsFlag,
 		true,
-		"enables automatic webhook installation (RBAC permission for service, secret and validatingwebhookconfigurations needed)",
+		"enables automatic certificates management for the webhook, the Secret and the ValidatingWebhookConfiguration must be created before running the operator",
```

Comment:
```suggestion
		"enables automatic certificate management for the webhook. the Secret and the ValidatingWebhookConfiguration must be created before running the operator",
```

the , -> . switch might make it a little clearer, I started parsing this as a list of resources that it manages